lib/class-wp-theme-json-resolver.php

```diff
@@ -287,8 +287,8 @@ public static function get_core_data() {
 	 */
 	public static function get_theme_data( $theme_support_data = array() ) {
 		if ( null === self::$theme ) {
-			$theme_json_data = self::get_from_file( locate_template( 'experimental-theme.json' ) );
-			self::translate( $theme_json_data, wp_get_theme()->get( 'TextDomain' ) );
+			$theme_json_data = self::read_json_file( self::get_file_path_from_theme( 'experimental-theme.json' ) );
+			self::translate_presets( $theme_json_data, wp_get_theme()->get( 'TextDomain' ) );
```

Comment:
Getting PHP fatal errors because of the calls to `translate_presets()` which no longer exists